src/main/java/gov/nih/nci/evs/api/service/DirectoryElasticLoadServiceImpl.java

```diff
@@ -62,21 +64,24 @@
 	@Autowired
 	ElasticOperationsService operationsService;
 
+	/**
+	 * @return filepath
+	 */
 	public File getFilepath() {
 		return filepath;
 	}
 
+	/**
+	 * @param filepath
+	 */
 	public void setFilepath(File filepath) {
 		this.filepath = filepath;
 	}
 
-	public int loadConcepts(ElasticLoadConfig config, Terminology terminology, HierarchyUtils hierarchy,
+	@Override
+  public int loadConcepts(ElasticLoadConfig config, Terminology terminology, HierarchyUtils hierarchy,
 			CommandLine cmd) throws Exception {
-		this.setFilepath(new File(cmd.getOptionValue('d')));
-		if (!filepath.exists()) {
-			throw new Exception("Given filepath does not exist");
-		}
-		try (final FileInputStream fis = new FileInputStream(this.getFilepath());
+		try (final FileInputStream fis = new FileInputStream(this.getFilepath() + "\\MRCONSO.RRF");
```

Comment:
Here you should definitely use "/" instead of "\\\\" because that should work on both Windows and Linux.